README.md

```diff
@@ -51,6 +51,7 @@ The following versions of Nginx should work with this module:
 
 Module version | Nginx version
 --- | ---
+1.3.0 | 1.19.2 or higher
```

Comment:
Does it have to be a specific version of `1.19.2` instead of `1.19.x`?

Reply:
testing it for 1.19.x (1.19.0, 1.19.1) as we speak, will post a fix in a few.

Reply:
it works for 1.19.x.
i've pushed a changed to the README.